monstereo/activity.py

```diff
@@ -121,6 +139,15 @@ def show_social(args, image_t, output_path, annotations, dic_out):
     # Prepare color for social distancing
     colors = ['r' if flag else 'deepskyblue' for flag in dic_out['social_distance']]
 
+    # Prepare color for raising hand with enough distance
```

Comment:
Many thanks for making the pull requests. Can you please make the raise hand feature completely separate from the social distancing? Because the former involves only one person at a time, the latter multiple people. Ideally something modulabile, where you can always build up more activities